api-reference/20 Data Visualization Widgets/dxSparkline/1 Configuration/type.md

```diff
@@ -9,199 +9,46 @@ default: 'line'
-            <li>showFirstLast</li>
-            <li>showMinMax</li>
-            <li>size</li>
-            <li>theme</li>
-            <li>tooltip</li>
-            <li>valueField</li>
-            <li>winColor</li>
-            <li>winlossThreshold</li>
-        </ul>
-    </td>
-  </tr>
-  </tbody>
-</table>
-</div>
-<script>
-$(function(){
-    $("#tableScrollView").dxScrollView({
-        showScrollBar: 'always',
-        direction: 'horizontal'
-    });
-})
-</script>
+The following options are available for all the **Sparkline** types:     
 
-To see the sparklines of each type in operation, refer to our [Visualization Gallery](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/SimpleSparklines).
+- [argumentField](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#argumentField)
+- [dataSource](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#dataSource)
+- [firstLastColor](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#firstLastColor)
+- [minColor](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#minColor)
+- [maxColor](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#maxColor)
+- [showFirstLast](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#showFirstLast)
+- [showMinMax](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#showMinMax)
+- [size](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/size/)
+- [theme](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#theme)
+- [tooltip](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/tooltip/)
+- [valueField](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#valueField)
+
+The following options are available for the **line**, **spline**, **stepline**, **area**, **splinearea**, and **steparea** types:      
+
+- [lineColor](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#lineColor)
+- [lineWidth](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#lineWidth)
+- [pointColor](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#pointColor)
+- [pointSize](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#pointSize)
+- [pointSymbol](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#pointSymbol)
+
+The following options are available for the **bar** type:
+
+- [barNegativeColor](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#barNegativeColor)
+- [barPositiveColor](/Documentation/ApiReference/Data_Visualization_Widgets/dxSparkline/Configuration/#barPositiveColor)
+
+The following options are available for the **winloss** type:
```

Comment:
We repeat this phrase four times. I think it would be better to make a list in which series types would be items and options would be subitems.